runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1220,8 +1220,15 @@ class INTERPRETER_CLASS
 		if (rc == (UDATA)obj) {
 			J9VMContinuation *continuation = _currentThread->currentContinuation;
 			if (NULL != continuation) {
-				/* Do not need to record whether contended monitor enter event is triggered after entered the monitor. */
-				continuation->runtimeFlags &= ~J9VM_CONTINUATION_RUNTIMEFLAG_JVMTI_CONTENDED_MONITOR_ENTER_RECORDED;
+				if (J9_ARE_ALL_BITS_SET(continuation->runtimeFlags, J9VM_CONTINUATION_RUNTIMEFLAG_JVMTI_CONTENDED_MONITOR_ENTER_RECORDED)) {
+					if (J9_EVENT_IS_HOOKED(_vm->hookInterface, J9HOOK_VM_MONITOR_CONTENDED_ENTERED)) {
+						J9ObjectMonitor *objectMonitor = monitorTableAt(_currentThread, obj);
+						ALWAYS_TRIGGER_J9HOOK_VM_MONITOR_CONTENDED_ENTERED(_vm->hookInterface, _currentThread, objectMonitor->monitor,
+							continuation->startTicks, J9OBJECT_CLAZZ(currentThread, obj), continuation->previousOwner);
```

Comment:
please make this formatting change after the PR builds succeed ... parameters need to start on a new line
```suggestion
						ALWAYS_TRIGGER_J9HOOK_VM_MONITOR_CONTENDED_ENTERED(
								_vm->hookInterface, _currentThread, objectMonitor->monitor,
								continuation->startTicks, J9OBJECT_CLAZZ(currentThread, obj), continuation->previousOwner);
```

Reply:
Updated.